lit.go

```diff
@@ -130,7 +226,16 @@ func main() {
 		log.Fatal(err)
 	}
 
-	litrpc.RpcListen(node, conf.rpcport)
+	rpcl := new(litrpc.LitRPC)
```

Comment:
Because the mutex for shutdown depends on `rpcl`. It was weird having litrpc create `rpcl` and then provide the lock for the whole program.